LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Input Output (Hong Kong) Ltd.
+   Copyright 2021 Input Output (Hong Kong) Ltd.
```

Comment:
```suggestion
   Copyright 2021-2023 Input Output (Hong Kong) Ltd.
```

Reply:
The same in other places, or am I missing something?

Reply:
I didn't realise there was a date in the `LICENSE` files.  I also found that the files sometimes disagreed, in which case I took the older date.

Reply:
I am pretty sure that the "APPENDIX: How to apply the Apache License to your work." section of the `LICENSE` is not supposed to be there (it looks to be an explanation for how to make the `NOTICE` file).

Reply:
I have removed the appendix!